src/System.Diagnostics.Process/src/System/Diagnostics/Process.Linux.cs

```diff
@@ -264,5 +271,26 @@ private Interop.procfs.ParsedStat GetStat()
             }
             return stat;
         }
+
+        /// <summary>Reads memory.limit_in_bytes cgroup data, if available.</summary>
+        /// <param name="limit">The read limit if available; otherwise, 0.</param>
+        /// <returns>true if the value could be read; otherwise, false.</returns>
+        private bool TryReadCGroupsMemoryLimit(out ulong limit)
+        {
+            const string CGroupMemoryLimitInBytesPath = "/sys/fs/cgroup/memory/memory.limit_in_bytes";
+            if (File.Exists(CGroupMemoryLimitInBytesPath))
+            {
+                string text = null;
+                try
+                {
+                    text = File.ReadAllText(CGroupMemoryLimitInBytesPath);
+                }
+                catch { } // ignore any failures reading the file
+                return ulong.TryParse(text, NumberStyles.Integer, CultureInfo.InvariantCulture, out limit);
```

Comment:
This can have multiplies suffix. This is what we do in CoreCLR to read this:

https://github.com/dotnet/coreclr/blob/030a3ea9b8dbeae89c90d34441d4d9a1cf4a7de6/src/pal/src/misc/sysinfo.cpp#L421

Reply:
Thanks.  Fixed.